eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -54,7 +54,6 @@ if ($null -eq [AzureEngSemanticVersion]::ParseVersionString($Version))
 $PkgProperties = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $ServiceDirectory
 $ChangeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $PkgProperties.ChangeLogPath
 
-
```

Comment:
looks like a left over change from your older update to this file that can be undone.

Reply:
The double space is not ideal either ways so I thought I'll just leave this.